### DISCUSSION TITLE
Allow users to disable window title from live timing window

### DISCUSSION THREAD

**Discussion by Jurtaani** [f1multiviewer/issue-tracker#97] [Feature requests]
as it is now, i can make the font smaller so i can fit all the 20 cars into that 1/4 sized box, in place of the official live timing video feed.
however that titlebar takes up space that i would much rather use for slightly bigger font or more race control messages.

**Comment by JJWatMyself** [comment]
Similar to 88. Suggest all vote on both.

https://github.com/f1multiviewer/issue-tracker/discussions/88

**Comment by jeppeolesen** [comment]
Redirecting #377 here. Relevant comment from that ticket here: https://github.com/f1multiviewer/issue-tracker/discussions/377#discussion-5136061
